exercises/concept/booking-up-for-beauty/.docs/hints.md

```diff
@@ -7,6 +7,7 @@
 ## 1. Parse appointment date
 
 - There is a [method][time.parse] for parsing a `string` into a `Time`.
+- Additional information for formatting parts of a date are in the [time format source][time.format.source]
```

Comment:
By source, you mean the comments in the source?

These comments are part of the documentation, as such, they also appear in the official documentation for the package: https://pkg.go.dev/time#pkg-constants

Linking to the docs would be better instead of the source code would be better here.

Reply:
Hi @andrerfcsantos , by source I mean the source code of the time format go file.

I've added this link because the comments at the top (lines 55-63) have exact notations for different parts of a date - year, month, day, hour, etc.

This is needed for declaring a layout when a string has to be formatted to a time layout.

Reply:
Right. What I'm saying is that the same comments are part of the documentation and are in the link I posted above. Linking to the docs is better than linking to source code.

Reply:
I agree that linking docs are better than source code, but that documentation lacks enough information to finish the exercise. The source code is slightly better in terms of proving info and examples.

Reply:
@andrerfcsantos  I think its better to copy relevant stuff from the comment section and add it to the exercise. Do you think its better to add it to the instruction or hints section ?

Reply:
Official docs for the time package constants contains a lot more info than whats asked in the exercise. Even minor formatting changes will produce a completely different time output. That's why I think relevant sections from the official documentation should be a part of the instruction. 

I have updated the link, but don't think it is a good idea. 

Learning to read the official docs is a good thing, but it would be appropriate to expect learners to go through the official docs in the next iteration of a learning exercise based on time.